src/pages/Services.js

```diff
@@ -157,4 +168,4 @@ function Services() {
     </>
   );
 }
-export default Services;
+export default Services;
```

Comment:
can you explain please why you removed this and again added the same code?

Reply:
i had copy pasted the code back from my saved file so it shows it like that

Reply:
okay